### PR TITLE
chore: bump components

### PR DESCRIPTION
**Pull request by billypchan**
bump for components
add new user type property form the mock user object

### CODE REVIEW

**Review by KaterinaWire — APPROVED** [review]

File: Tests/Source/Data Model/ServiceUserTests.swift [hunk 1]
```diff
@@ -23,6 +23,8 @@ import XCTest
 
 final class DummyServiceUser: NSObject, ServiceUser {
     
+    var isTrusted: Bool = true
```
Comment (typfel) [inline Tests/Source/Data Model/ServiceUserTests.swift:26]: Shouldn't trusted be false?

**Review by typfel — CHANGES_REQUESTED** [review]

File: Tests/Source/Data Model/ServiceUserTests.swift (hunk 1 above)
Comment (billypchan) [reply Tests/Source/Data Model/ServiceUserTests.swift:26]: I ran the full test suite and it passes, but I can also set it to false.

Comment (typfel) [reply Tests/Source/Data Model/ServiceUserTests.swift:26]: Please do I think that reflects reality more

Comment (billypchan) [reply Tests/Source/Data Model/ServiceUserTests.swift]: tested with false and it is passed

**Review by typfel — APPROVED** [review]

**Comment by billypchan** [comment]
@zenkins release patch

**Pull request merged**